src/components/NavFooter.js

```diff
@@ -6,9 +6,26 @@ import {
   Icon,
   useTranslation,
 } from '@newrelic/gatsby-theme-newrelic';
+import ctaJson from '../data/nav-footer-cta';
 
 const NavFooter = ({ className }) => {
   const { t } = useTranslation();
+  let ctaContent = ctaJson.default;
+  const allCTAs = Object.keys(ctaJson);
+
+  if (typeof window !== 'undefined') {
+    const currentPage = location.pathname;
+
+    for (const product of allCTAs) {
+      if (
+        currentPage.includes(ctaJson[product].directory) &&
+        product !== 'default'
+      ) {
+        ctaContent = ctaJson[product];
+      }
+    }
+  }
```

Comment:
if you wanna remove the window check you could use the useLocation hook from reach/router here

Reply:
brilliant. thank you. i knew that didn't feel right.